recharges/models.py

```diff
@@ -1,19 +1,18 @@
 import uuid
 
-from django.contrib.postgres.fields import HStoreField
 from django.db import models
 
 
-class DummyModel(models.Model):
+class Recharge(models.Model):
 
     """
-    - Nice docstring please
+    Stores recharge entries
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    product_code = models.CharField(max_length=20)
-    data = HStoreField(null=True, blank=True)
+    amount = models.DecimalField(max_digits=9, decimal_places=2)
+    msisdn = models.CharField(max_length=20)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):  # __unicode__ on Python 2
-        return str(self.id)
\ No newline at end of file
+        return "R%s recharge for %s" % (self.amount, self.msisdn)
```

Comment:
Drop the `R` please as that assumes currency of Rand which isn't defined in the system yet